luaui/Widgets/gui_nanotrans_range.lua

```diff
@@ -0,0 +1,97 @@
+function widget:GetInfo()
+    return {
+       name         = "Nano range on transport",
+       desc         = "Draw a circle around a transport carrying a nano when its about to unload",
+       author       = "Cheva",
+       date         = "December 2024",
+       license      = "GNU GPL, v2 or later",
+       layer        = 0,
+       enabled      = true
+    }
+end
+
+--------------------------------------------------------------------------------
+--vars
+--------------------------------------------------------------------------------
+local circleDivs = 96
+local range
+local isNanoTC = {}
+local turretRange = {}
+local transportWithNano = {}
+
+--------------------------------------------------------------------------------
+--speedups
+--------------------------------------------------------------------------------
+local CMD_UNLOAD_UNITS = CMD.UNLOAD_UNITS
+local spGetActiveCmd = Spring.GetActiveCommand
+local GetSelectedUnitsSorted = Spring.GetSelectedUnitsSorted
+local glColor = gl.Color
+local glLineWidth = gl.LineWidth
+local glDrawGroundCircle = gl.DrawGroundCircle
+
+for udid, ud in pairs(UnitDefs) do
```

Comment:
```suggestion
for unitDefId, unitDef in pairs(UnitDefs) do
```
Avoid short forms

Reply:
Changed the var names to more descriptive form